test/flash.test.js

```diff
@@ -66,46 +67,45 @@ const unfakeSTO = function() {
   window.setTimeout = oldSTO;
 };
 
-const MockSegmentParser = function() {
-  let ons = {};
-  let datas = [];
-
-  this.on = function(type, fn) {
-    if (!ons[type]) {
-      ons[type] = [fn];
-    } else {
-      ons[type].push(fn);
-    }
-  };
-  this.trigger = function(type, data) {
-    if (ons[type]) {
-      ons[type].forEach(function(fn) {
-        fn(data);
-      });
-    }
-  };
-  this.getFlvHeader = function() {
-    return new Uint8Array([1, 2, 3]);
-  };
-
-  this.push = function(data) {
-    datas.push(data);
-  };
-  this.flush = function() {
-    let tags = datas.reduce(function(output, data, i) {
-      output.push(makeFlvTag(i, data));
-      return output;
-    }, []);
-
-    datas.length = 0;
-    this.trigger('data', {
-      tags: {
-        videoTags: tags,
-        audioTags: []
+// Create a WebWorker-style message that signals the transmuxer is done
+const createDataMessage = function(datas, audioDatas, metadata, captions) {
```

Comment:
datas sounds funny to me. Maybe dataArray and audioDataArray? Or just data and audioData and it is implied it is an array?